client/modules/IDE/actions/projects.unit.test.js

```diff
@@ -33,9 +33,9 @@ describe('projects action creator tests', () => {
     store = mockStore(initialTestState);
 
     const expectedActions = [
-      { type: ActionTypes.START_LOADING },
+      { type: 'loading/startLoader' },
```

Comment:
Either of these should work:
```
{ type: startLoader.type },
```
```
startLoader(),
```
`startLoader()` is an action creator, so it's a function that returns the action.  If you just call that function without dispatching you will get the value `{ type: 'loading/startLoader' }`.